src/cascadia/TerminalApp/pch.h

```diff
@@ -76,7 +76,5 @@ TRACELOGGING_DECLARE_PROVIDER(g_hTerminalAppProvider);
 
 #include <winrt/Windows.UI.Popups.h>
 
-#include <CLI11/CLI11.hpp>
-

```

Comment:
Shouldn't we reference the CommandlineArgs lib? Then we can remove the references to `../CommandlineArgs/AppCommandlineArgs.h` in TerminalPage.h and CommandPalette.h?

Reply:
Oh but the commandline args lib isn't a winrt project. So there's not a `winrt/Microsoft.Terminal.CommandlineArgs` generated header that's particularly punishing to include outside the pch.